introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-chatbot-openchatkit.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cmd_queue = [
</pre><p>what does the shell looks like? any chance we include a screencut to show how nice it is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I plan to include this in a related blog post -- would prefer to avoid merging binary files here if possible.